src/Blockcore.Indexer/wwwroot/css/bootstrap.min.css

```diff
@@ -0,0 +1,6 @@
+/*!
+ * Bootstrap v3.3.7 (http://getbootstrap.com)
```

Comment:
Not sure I like the idea of including bootstrap just for this very basic status page? It's also a very old version of bootstrap, should at least be latest.

Reply:
I do not agree that I used Bootstrap. I will remove it without changing the current appearance.

Reply:
If you could do the layout without bootstrap, then that would be awesome! Thanks =) <3

Reply:
Check now. It got much better

Reply:
![firefox_oIColC05DF](https://user-images.githubusercontent.com/6504337/158036791-c332049d-dd8a-4494-9562-7d50b45072a0.png)

Reply:
![image](https://user-images.githubusercontent.com/6504337/158036810-2a17a227-1af1-4b9d-b176-7e521102da78.png)